cartridges/openshift-origin-cartridge-haproxy/bin/control

```diff
@@ -147,15 +147,19 @@ function _stop_haproxy_service() {
     else
         if `pgrep -x haproxy > /dev/null 2>&1`
         then
-            echo "Warning: HAProxy process exists without a pid file.  Use force-stop to kill." 1>&2
+            client_message "Warning: HAProxy process exists without a pid file."
```

Comment:
Right. The closest thing we have is `client_debug`, but I don't think that is appropriate here.